<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbtn_start_download">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>开始下载</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>Paper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Forge</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vanilla</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bukkit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spigot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mojang</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nukkit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CubeRite</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GlowStone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Akarin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BunggeeCord</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BukkitPlugins</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TacoSpigot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cauldron</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thermos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WaterFall</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>1.18.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.17.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.16.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.16.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.16.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.16.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.16.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.15.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.15.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.14.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.14.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.14.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.14.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.13.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.13.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.12.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.12.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.11.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.11.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.10.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.10.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.9.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.9.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.9.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.9.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.8.9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.8.8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.8.7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.8.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.8.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.8.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.8.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.8.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.8.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.7.9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.7.8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.7.7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.7.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.7.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.7.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.7.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.7.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.7.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.7</string>
    </property>
   </item>
  </widget>
  <widget class="QTextBrowser" name="tb_about">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;###############################################&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;         本列表取自SakuraMirror,并非MSL2(C#)所使用的初梦资源站&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;     此列表包含以下种类服务端：使用Java的JE服务端          				非Java的JE服务端             				使用C++的BE服务端&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;                                         	使用Java的BE服务端&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;                                         	带反向代理的服务端&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;		互通服服务端&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;                      注意：有部分服务端只有老版本（大部分时候是1.12.2版本）&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;             部分服务端的版本与游戏版本不一样，此类服务端没有标注是否停更&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;						使用[B]表示BE版服务端，除此之外基本都是JE版服务端&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;						使用[P]表示带反向代理的服务端，除此之外基本都不带反向代理&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;						使用[J]表示使用Java的BE服务端&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;						使用[C]表示使用C++的JE服务端&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;						使用[版本号]表示目前只有老版本的服务端&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;						使用[H]表示比较硬核，新手不建议使用的服务端，通常情况下会与[P]一起出现&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;						使用[M]表示性能较强的服务端&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;								&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;							整理不易，望采纳！&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;###############################################&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.12.2]Akarin&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Akarin 是一款支持多线程的 Minecraft 服务端，来自 “新维度”，前身是 TorchSpigot 服务端。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[P][H][M]BungeeCord&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;BungeeCord 是一个高性能的反向代理服务端，它可以将多个 Minecraft 服务器变成一个 “群组服务器”。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;使用介绍：http://www.mcbbs.net/thread-424117-1-1.html&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.7.10]Contigo&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Contigo 是 Thermos 的一个分支版本，支持 Mod 和插件。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.12.2]Craft Bukkit&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Craft Bukkit 是一个老牌的 Minecraft 服务端，支持大部分插件，功能比较完善，目前是更新比较快的几个服务端之一。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;这个服务端容易出现一些性能问题，如果你的服务器经常在日志中出现“Can't keep up”之类的报错，可以考虑使用 Spigot、PaperSpigot 等服务端。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[C][M]CubeRite 是一个基于 C++ 编写的开源高性能 Minecraft 服务端&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;目前 Cuberite 已经可以做到大部分的基于 Bukkit 架构的 Minecraft 服务端（例如 Spigot）的功能，并且在性能方面具有更大的优势。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[P][H][M]DragonProxy&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;DragonProxy 龙代理是一个高性能的反向代理服务端，其功能类似于 BungeeCord，但是它的作用是让电脑和手机一起联机&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.12.2][P][H]GlowStone&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;GlowStone 萤石是一款开源的 Bukkit 服务端，开发者可以根据自己需求修改或制作一个服务端，内置了 Sponge 支持的插件。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[P][H][M]HexaCord&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;HexaCord 是一个 fork 自 BungeeCord 的反向代理服务端，基于原基础做了一些优化，包括性能的提升。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.11.2][M]Hose&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Hose 是一款开源的多线程 Minecraft 服务端，性能更强。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[B]MiNET&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;MiNET 是一个用 C 语言编写的 Minecraft PE 基岩版服务端。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;MiNET 面向的是大型的 Minecraft 游戏服务器，拥有企业级的稳定性。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;服务端的每个实例可以同时在线 10-100 个玩家，MiNET 的目标是千人在线（很强）。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;MiNET 在未来可能会成为微软云产品（官方原话翻译）&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Bugjump&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;懂得都懂，官方服务端，啥都加不了&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[B][J]Nukkit&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Nukkit 是一款由 Java 驱动的 Minecraft PE 携带版服务端，适合开一个多人联机的 PE 版本服务器。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[J]NukkitX&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;NukkitX 应该是 Nukkit 的强化 / 后续版本，这是一个高性能的 Minecraft PE 基岩版服务器，由 Java 强力驱动。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.12.2]PaperSpigot&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Spigot 的进一步优化版本，在相关算法方面，较 Spigot 有所提高，优化 TPS 等，支持 CraftBukkit 和 Spigot 插件，API 没有太大修改。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;在 1.9 版本之前都带有反作弊功能，1.9 之后的版本需要自行安装其他反作弊插件。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;PocketMine-MP&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;PocketMine-MP 是一个用 PHP 开发的 Minecraft Bedrock 基岩版服务端，也是目前使用最为广泛的服务端之一。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;因为高可扩展性和插件编写起来相对简易，深受广大手机版服主喜爱。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.13.2]Spigot&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Spigot 是 CraftBukkit 服务端之后的延续版本，比 CraftBukkit 优化了不少地方，支持 CraftBukkit 的插件，性能比 CraftBukkit 好很多，并且自带反作弊功能&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.11.2]Sponge Forge&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Sponge 是一个全新的服务端，支持 Sponge 的专用插件，可装 Mod，兼容性比 Cauldron 相比提高了不少，适合开 MOD 服，支持的版本非常高，是目前支持 MOD 的服务端里兼容版本最高的服务端。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Sponge 本身不支持 Bukkit 插件（即使有兼容层，效果也不是很好，只能支持一般的插件），需要服务器的配置比较高，启动速度不佳。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.12.2]Sponge Vanilla&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;SpongeVanilla 是 SpongeForge 的纯净版本，同样支持 SpongeForge 的插件。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;性能相对较好，占用的内存较少，适合低配机器开服，对 Bukkit 插件的支持不佳，插件非常少，配置不方便，不能安装Mod。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.12.2]TacoSpigot&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;TacoSpigot 是 Spigot 的优化版本，支持 CraftBukkit 和 Spigot 的插件，API 基本无变化，自带反作弊。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.12.2]Torch&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Torch 是一款由国人制作的服务端，是 Spigot 的优化版本，支持非常多插件，性能强悍，支持多核心 CPU 运算，自带反作弊功能。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Vefland 是由格兰蒂协会创始人乐乐接手 TorchSpigot 开发后的新项目，同样是一个多线程的服务端，性能优化极佳，启动速度比香港记者还快。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[1.12.2]Velocity&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Velocity 是一个 Minecraft 反向代理服务端，拥有非常强大的服务器支持，可扩展性和灵活性。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;对于大型群组服务器，Velocity 是一个很好的选择。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;WaterFall&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;WaterFall 是 BungeeCord 的一个分支。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Waterfall 致力于优化 Bungeecord 在代理服务端数据过程中产生的不必要垃圾，使你的 BungeeCord 性能更高，更充分发挥在很高玩家在线数时 CPU 的性能。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;大家都知道，一旦 MCPC，或者 KCauldron 等 mod 服务端在 BC 进行跨服的时候，这将会使尝试跨服的玩家的客户端崩溃，而 WaterFall 不会,它及时修复了这个 Bug。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
